### PR TITLE
Added Chat configuration 

### PR DESCRIPTION
**Pull request by tarelmadar**
Reverts beyond-io/sellotape#69

### CODE REVIEW

File: sellotape/sellotape_dj/static/reconnecting-websocket.js [hunk 1]
```diff
@@ -0,0 +1,382 @@
+// MIT License:
+//
+// Copyright (c) 2010-2012, Joe Walnes
```
Comment (ItzhakiGuy) [inline sellotape/sellotape_dj/static/reconnecting-websocket.js:3]: I am not familiar with that project, 
Why is it relevant to the chat application?
Maybe it should be on a separate PR for stating its purpose?

Comment (tarelmadar) [reply sellotape/sellotape_dj/static/reconnecting-websocket.js:3]: This is a JS file that handles the messaging.
please read the documentation in it - it seems to be very clear.

Comment (ItzhakiGuy) [reply sellotape/sellotape_dj/static/reconnecting-websocket.js:3]: It is a JS project for web socketing, however, it doesn't state how it has relevance to our app, nor what it is supposed to do on chats.
To make it clearer, I was asking about it and suggesting to upload a followed PR with a relevant explanation on the subject.

Comment (tarelmadar) [reply sellotape/sellotape_dj/static/reconnecting-websocket.js:3]: Again, the chat app can't run without reconnecting-websocket.js. 
You can read the documentation inside the file for more understanding.
besides if it still not clear please find attached the readme file at this link -https://github.com/joewalnes/reconnecting-websocket.

Comment (ItzhakiGuy) [reply sellotape/sellotape_dj/static/reconnecting-websocket.js:3]: There is a difference between running and describing functionality, my question was asked to understand why that project was needed for the app. the documentation of this project described that the library is a web socket connection handler, however, could you elaborate on its use in the chat app?

Comment (tarelmadar) [reply sellotape/sellotape_dj/static/reconnecting-websocket.js:3]: I already shared with you the files that explain your questions. 
let's take it offline if you have any more questions.

File: sellotape/sellotape_dj/sellotape_dj/routing.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from channels.auth import AuthMiddlewareStack
```
Comment (11galpid) [inline sellotape/sellotape_dj/sellotape_dj/routing.py:1]: Note, you fail the tests:

ModuleNotFoundError: No module named 'channels'
Error: Process completed with exit code 1.

Please open a PR that adds the 'channels' module before pushing this PR, so we can see that the tests check out

**Review by 11galpid — CHANGES_REQUESTED** [review]

File: sellotape/sellotape_dj/sellotape_dj/routing.py (hunk 2 above)
Comment (tarelmadar) [reply sellotape/sellotape_dj/sellotape_dj/routing.py:1]: I've done that, tests still going wrong.
can you please check again?

**Review by rosomri — APPROVED** [review]
This PR includes just the configuration and doesn't include new tests,
the Settings, Routing and the URLs look good.
Just make sure that the other PR (#70) tests also the reconnecting-websocket. 
All the previous tests passed so I approve this PR.

**Review by 11galpid — APPROVED** [review]
Nice
